go/client/tree_test.go

```diff
@@ -112,6 +116,242 @@ func TestBuildTree(t *testing.T) {
 	}
 }
 
+type inputPath struct {
+	path          string
+	fileContents  []byte
+	isSymlink     bool
+	isAbsolute    bool
+	symlinkTarget string
+}
+
+func construct(dir string, ips []*inputPath) error {
+	for _, ip := range ips {
+		path := filepath.Join(dir, ip.path)
+		if ip.isSymlink {
+			target := ip.symlinkTarget
+			if ip.isAbsolute {
+				target = filepath.Join(dir, target)
```

Comment:
Where does `target` get used?

Reply:
Oh! Lol, it didn't, thank you! It was a bug! Now my test case of absolute symlinks actually tests this path.